dcos/http.py

```diff
@@ -83,7 +83,8 @@ def request(method,
             request.headers)
 
         with requests.Session() as session:
-            response = session.send(request.prepare(), timeout=timeout)
+            response = session.send(request.prepare(), timeout=timeout,
+                                    proxies=util.get_proxy_dict_from_env())
```

Comment:
@mgummelt I suspect the issue they ran into was case sensitivity. On Windows I think environment variables are case insensitive and on Linux they are case sensitive.